configuration/src/main/java/com/cerner/beadledom/configuration/EnvironmentConfigurationSource.java

```diff
@@ -0,0 +1,68 @@
+package com.cerner.beadledom.configuration;
+
+import org.apache.commons.configuration2.Configuration;
+import org.apache.commons.configuration2.EnvironmentConfiguration;
+
+/**
+ * An implementation class of {@link ConfigurationSource} for the source of environment variables.
```

Comment:
would it be a good idea to link environment variables to this https://docs.oracle.com/javase/tutorial/essential/environment/env.html ?

Reply:
I think it would be. Added, https://github.com/cerner/beadledom/pull/55/commits/adde21acd9b2ebf94119f57d20551f36f1e5ad50